src/odemis/gui/cont/fastem_acq.py

```diff
@@ -512,14 +512,23 @@ def on_acquisition(self, evt):
         # Acquire ROAs for all projects
         fs = {}
         pre_calibrations = [Calibrations.OPTICAL_AUTOFOCUS, Calibrations.IMAGE_TRANSLATION_PREALIGN]
+        pre_calib_plus = [Calibrations.OPTICAL_AUTOFOCUS,
+                          Calibrations.AUTOSTIGMATION,
+                          Calibrations.IMAGE_TRANSLATION_PREALIGN]
+
+        # Read the frequency with which to run autostigmation from the acquisition configuration
+        acqui_conf = conf.get_acqui_conf()
+        autostig_freq = math.inf if acqui_conf.autostig_freq == 0 else acqui_conf.autostig_freq
```

Comment:
Period is indeed a better option, it is every how many sections you need to run the autostigmation. So for the value 3, it will run autostigmation every 3 sections